app/Http/Controllers/UserController.php

```diff
@@ -56,6 +56,7 @@ public function index()
         return view('user.index', [
             'users' => User::orderBy('username')->get(),
```

Comment:
Update this to:
`'users' => User::with('preferences')->orderBy('username')->get(),`

and delete the 'twofactor' line below.